content/guides/comparators.adoc

```diff
@@ -0,0 +1,635 @@
+= Comparators in Clojure
+Andy Fingerhut
+2013-04-01
+:jbake-type: page
+:toc: macro
+
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+
+== Versions
+
+This document was written while checking the details against Clojure 1.5.1  and Java 6,
```

Comment:
We should write this from the perspective of Clojure 1.7 (the current stable version).